client/components/payments-activity/payments-data-tile.tsx

```diff
@@ -3,44 +3,16 @@
  */
 import * as React from 'react';
 import { __ } from '@wordpress/i18n';
+import { Link } from '@woocommerce/components';
 
 /**
  * Internal dependencies
  */
-import './style.scss';
+
 import { formatCurrency } from 'wcpay/utils/currency';
 import Loadable from '../loadable';
-import { Link } from '@woocommerce/components';
-interface PaymentsDataTileProps {
-	/**
-	 * The id for the tile, can be used for CSS styling.
-	 */
-	id: string;
-	/**
-	 * Label for the amount in the tile.
-	 */
-	label: string;
-	/**
-	 * The currency code for the amount displayed.
-	 */
-	currencyCode: string;
-	/**
-	 * For optionally passing a ClickTooltip component.
-	 */
-	tooltip?: React.ReactElement;
-	/**
-	 * The amount to be displayed in the tile.
-	 */
-	amount?: number;
-	/**
-	 * Loading state of the tile.
-	 */
-	isLoading?: boolean;
-	/**
-	 * Optional hover link to view report.
-	 */
-	reportLink?: string;
-}
+import { PaymentsDataTileProps } from './interfaces';
```

Comment:
💭 I'm curious why this was moved to a separate file. It seemed a more consistent pattern to keep the props type definition alongside the component definition.